.gitignore

```diff
@@ -114,4 +114,10 @@ examples/resources
 dist
 
 # temporary data
-temporary.zip
\ No newline at end of file
+temporary.zip
+
+# vscode
+.vscode/
+
+# DS_Store
+.DS_Store
```

Comment:
@mski-iksm This kind of ignore entries should be set in personal git configurations (not in specific repo's config).

Reply:
ex) personal environment specific configurations
https://github.com/github/gitignore/tree/main/Global

IMO, this kind of ignore entries put in `$HOME/.config/git/ignore`
ref: https://git-scm.com/docs/gitignore

Reply:
@hirosassa 
Thank you for the comment. I see how it should be managed.

I will close this PR.